cl/compile.go

```diff
@@ -785,9 +786,18 @@ func loadFuncBody(ctx *blockCtx, fn *gox.Func, body *ast.BlockStmt) {
 	cb.End()
 }
 
+func simplifyGopPackage(pkgPath string) string {
+	match, _ := regexp.MatchString("^gop/(\\w+(/)*)+$", pkgPath)
```

Comment:
maybe we just need `strings.HasPrefix(pkgPath, "gop/)"`?

Reply:
I agree, it's better this way.